src/api/config/personalAttributionsConfig.js

```diff
@@ -0,0 +1,142 @@
+// @flow
+
+import {
+  type IdentityId,
+  type Name,
+  identityIdParser,
+  nameParser,
+} from "../../core/identity";
+import {type Ledger} from "../../core/ledger/ledger";
+import {
+  type TimestampISO,
+  timestampISOParser,
+  fromISO,
+} from "../../util/timestamp";
+import * as C from "../../util/combo";
+import {type PersonalAttributions} from "../../core/credrank/personalAttribution";
+
+/**
+  This type resembles the JSON schema for configuring personal attributions,
+  which allows participants to attribute their cred to other participants.
+  This feature should not be used to make cred sellable/transferable, but
+  instead is intended to allow participants to acknowledge that a portion of
+  their creditted outputs are directly generated/supported by the labor of
+  others. (e.g. when a contributor has a personal assistant working behind the 
+  scenes)
+ */
+export type PersonalAttributionsConfig = Array<{|
+  // The name of a participant that is flowing cred to others
+  fromParticipantName: Name,
+  // The id of the participant. Will be autopopulated if absent.
+  fromParticipantId?: IdentityId,
+  +recipients: Array<{|
+    // The name of a participant that is receiving cred
+    // from the "fromParticipant"
+    toParticipantName: Name,
+    // The id of the participant. Will be autopopulated if absent.
+    toParticipantId?: IdentityId,
```

Comment:
Same comment as above. It might make sense to write a `participantValidator` or `participantUpdater` function that updates this and the `from` values according to the logic above.

Reply:
Literally what updatePersonalAttributionsConfig does.